mpc-spec/src/utils.rs

```diff
@@ -0,0 +1,65 @@
+//! Utility functions for the MPC specification.
+
+use crate::party::ChannelConfig;
+use std::sync::mpsc::{channel, Receiver, Sender};
+
+use crate::messages::MPCMessage;
+pub mod rand;
```

Comment:
maybe move this to the `use` items above?

Reply:
:+1: That's what I get for forgetting to run `cargo fmt` again, sorry.